powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/errorhandling/DefaultExceptionHandler.java

```diff
@@ -48,15 +48,27 @@ public class DefaultExceptionHandler {
     }
 
     /**
-     * Exception handler for application not found exception.
+     * Exception handler for application configuration not found exception.
      * @param ex Exception.
      * @return Response with error details.
      */
     @ExceptionHandler(AppConfigNotFoundException.class)
     @ResponseStatus(HttpStatus.BAD_REQUEST)
-    public @ResponseBody ErrorResponse handleApplicationNotFoundException(AppConfigNotFoundException ex) {
-        logger.warn("Error occurred during application lookup.", ex);
-        return new ErrorResponse("APPLICATION_NOT_FOUND", "Application was not found.");
+    public @ResponseBody ErrorResponse handleAppConfigNotFoundException(AppConfigNotFoundException ex) {
+        logger.warn("Error occurred during application configuration.", ex);
+        return new ErrorResponse("APP_CONFIG_NOT_FOUND", "Application configuration was not found.");
```

Comment:
🤔 We are changing the error code, but it was completely wrong anyway. We should document the values. Opening an issue for that #280

Reply:
Yes, we should document more than just error codes.